pkgs/development/libraries/mesa/default.nix

```diff
@@ -129,6 +129,9 @@ self = stdenv.mkDerivation {
     "-Dgallium-nine=${boolToString enableGalliumNine}" # Direct3D in Wine
     "-Dosmesa=${boolToString enableOSMesa}" # used by wine
     "-Dmicrosoft-clc=disabled" # Only relevant on Windows (OpenCL 1.2 API on top of D3D12)
+
+    # To enable non-mesa gbm backends to be found (e.g. Nvidia)
+    "-Dgbm-backends-path=${libglvnd.driverLink}/lib/gbm:${placeholder "out"}/lib/gbm"
```

Comment:
since glvnd is listed under the linux only section, should the libglvnd backend  also be conditionally added?

Reply:
I'm not really sure, I just copied this pattern from the analogous dri search paths option above.

Reply:
(aka, I don't think it's any more wrong than what's already in place... for what that's worth.)

Reply:
also, since it's a search path, I can't see it causing harm.

Reply:
I'm not sure if `driverLink` is really meaningful on `*-darwin`.

Reply:
In which case, this won't hurt it. We don't conditionalize the use of `driverLink` for dri-paths.

If it's really important to do right for GBM, or fix both of these... I can look into just not setting the DRI/GBM search paths on `!isLinux`,,,

Reply:
May increase closure size unnecessarily on darwin, would be my guess?

Reply:
So again, it seems like the ask without being stated is for me to fix this retroactively, since this is already in use for the dri search paths. My change as-is will not affect closure size.

Reply:
Sorry, I wasn't aware that it already unconditionally includes `libglvnd.driverLink`. If the closure size isn't increased, then there's no need for intervention in this PR, imo.

Reply:
ah, if it's already in the closure, then it doesn't really matter